netbench/netbench/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "netbench"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
we also do not publish this crate `publish = false` so dont need to bump version. Could you add the no bump comment here instead.